src/models/StyleSheet.js

```diff
@@ -105,6 +106,10 @@ export default class StyleSheet {
     return this.tags.map(tag => tag.toHTML()).join('')
   }
 
+  toElement() {
+    return this.tags.map(tag => tag.toElement())
```

Comment:
Hm yes but what should I use for keys? `this.names.join('')` in ServerTag.toElement?